packages/components/src/FormField/FormField.test.tsx

```diff
@@ -366,6 +366,16 @@ test("it should set the inputMode when the keyboard prop is set", () => {
   expect(name).toContain(keyboardMode);
 });
 
+test("it should render the spinner when loading is true", () => {
+  const { getByLabelText } = render(
+    <FormField placeholder="foo" type="text" loading={true} />,
+  );
+  const spinner = getByLabelText("loading");
+  const classNames = spinner.getAttribute("class");
+
+  expect(classNames).toContain("spinner small");
```

Comment:
I think we can just check chat the result of `getByLabelText("loading")` is an instance of HTMLElement as the rest of this is checking concerns that are internal to the Spinner component.